web/src/presence.ts

```diff
@@ -133,6 +133,28 @@ export function status_from_raw(raw: RawPresence): PresenceStatus {
 
     last_active = active_timestamp ?? idle_timestamp;
 
+    /*
+        We want to show the last idle_timestamp as the last_active
+        for the user who has turned on the 'Go invisible' mode.
+        For this, we need to compare the difference between the date_joined
+        and last_active time of the user. If it's less than the offline_threshold_secs,
+        we can consider the user has gone invisible and we can show
+        the idle_timestamp as the last_active time.
+    */
+
+    const user = people.get_by_user_id(user_id);
+    if (last_active && user.date_joined) {
+        const dateJoined = new Date(user.date_joined).getTime() / 1000;
```

Comment:
Please don't submit PRs with code using camelCase variables. If you're using an AI tool that generates such code, you should either stop using the tool or learn how to review the code yourself.